docs/reference-architectures/sharepoint/index.md

```diff
@@ -111,143 +116,264 @@ Before configuring the SharePoint farm, make sure you have one Windows Server AD
 
-In addition, it's always wise to plan for security hardening. Other recommendations include:
+- Don’t assign public IP addresses to VMs.
 
-- Add rules to NSGs to isolate subnets and roles.
-- Don't assign public IP addresses to VMs.
 - For intrusion detection and analysis of payloads, consider using a network virtual appliance in front of the front-end web servers instead of an internal Azure load balancer.
+
 - As an option, use IPsec policies for encryption of cleartext traffic between servers. If you are also doing subnet isolation, update your network security group rules to allow IPsec traffic.
+
 - Install anti-malware agents for the VMs.
 
-## Deploy the solution
+## Community
+Communities can answer questions and help you set up a successful deployment. Consider the following:
+
+- [Azure Forum][azure-forum]
+
+- [TechNet Wiki: SharePoint: Community Best Practices][technet-wiki]
+
+- [SharePoint Microsoft Tech Community][sharepoint-community]
+
+## Solution deployment
+
+The deployment scripts for this reference architecture are available on [Github][github]. To run the PowerShell script that deploys this architecture, use the latest version of the Azure [command line interface][azure-cli] (CLI).
+
+You can deploy this architecture incrementally or all at once. The first time, we recommend an incremental deployment so you can see what each deployment does. Specify the increment using one of the following *mode* parameters, which controls the granularity of the deployment:
+
+| Mode           | What it does                                                                                                            |
+|----------------|-------------------------------------------------------------------------------------------------------------------------|
+| onprem         | Deploys a simulated on-premises network environment. This option does not connect to any existing on-premises networks. |
+| infrastructure | Deploys the SharePoint 2016 network infrastructure and jumpbox in Azure.                                                |
+| createvpn      | Deploys the virtual network gateway for both the SharePoint and on-premises networks and connects them.                 |
+| workload       | Deploys the SharePoint servers to the SharePoint network.                                                               |
+| security       | Deploys the network security group to the SharePoint network.                                                           |
+| all            | Deploys all the preceding deployments.                                                                                  |
+
+> [!NOTE]
+>  We recommend deploying this architecture incrementally in the order shown (infrastructure first, then applications). If you don’t have an on-premises network and want to deploy the entire reference architecture for testing and evaluation, specify **all** for the mode and note that process may take several hours.
+>
```

Comment:
These statements seem slightly contradictory:

- The first time, we recommend an incremental deployment so you can see what each deployment does. 
- We recommend deploying this architecture incrementally in the order shown

Sounds like we’re saying you should mostly use ‘incremental’ unless you want a test deployment?